Description: Fix for GCC >= 4.7
 Patch is already available in master branch in upstream.
Author: Boris Pek <tehnick-8@yandex.ru>
Last-Update: 2012-05-10

--- a/eiskaltdcpp-qt/src/PublicHubs.cpp
+++ b/eiskaltdcpp-qt/src/PublicHubs.cpp
@@ -151,7 +151,7 @@ void PublicHubs::slotContextMenu(){
         return;
 
     if (proxy)
-        std::transform(indexes.begin(), indexes.end(), indexes.begin(), [&proxy](QModelIndex i) { return proxy->mapToSource(i); });
+        std::transform(indexes.begin(), indexes.end(), indexes.begin(), [&](QModelIndex i) { return proxy->mapToSource(i); });
 
     WulforUtil *WU = WulforUtil::getInstance();
 
